Tools/jhbuild/jhbuild-minimal.modules

```diff
@@ -224,4 +226,32 @@
     </dependencies>
   </meson>
 
+  <!-- libjxl required for JPEGXL support -->
+  <cmake id="libjxl"
+         cmakeargs="-DBUILD_TESTING=OFF
+                    -DPROVISION_DEPENDENCIES=ON
+                    -DJPEGXL_ENABLE_FUZZERS=OFF
+                    -DJPEGXL_ENABLE_DEVTOOLS=OFF
+                    -DJPEGXL_ENABLE_TOOLS=OFF
+                    -DJPEGXL_ENABLE_MANPAGES=OFF
+                    -DJPEGXL_ENABLE_BENCHMARK=OFF
+                    -DJPEGXL_ENABLE_EXAMPLES=OFF
+                    -DJPEGXL_ENABLE_JNI=OFF
+                    -DJPEGXL_ENABLE_VIEWERS=OFF
+                    -DJPEGXL_ENABLE_TCMALLOC=OFF
+                    -DJPEGXL_ENABLE_PLUGINS=OFF
+                    -DJPEGXL_ENABLE_COVERAGE=OFF
+                    -DJPEGXL_ENABLE_TRANSCODE_JPEG=OFF
+                    -DJPEGXL_ENABLE_SJPEG=OFF
+                    -DJPEGXL_STATIC=OFF
+                    -DJPEGXL_WARNINGS_AS_ERRORS=OFF
+                    -DJPEGXL_ENABLE_SKCMS=ON">
+    <branch module="libjxl/libjxl.git"
+            tag="v0.6.1"
+            checkoutdir="libjxl-0.6.1"
+            repo="github.com">
+      <patch file="libjxl-add-cmake-flag-provision-dependencies.patch" strip="1"/>
```

Comment:
In order to tell jhbuild to only build this dep if is not on the system we need to specify a version and a pkg-config filename. Otherwise jhbuild will build always the lib even when the system already has it.
The idea of the minimal jhbuild is to only build libraries is strictly needed.

Add this:
```
<cmake id="libjxl" ....
 <pkg-config>libjxl.pc</pkg-config>
 <branch module="libjxl/libjxl.git" version="0.6.1" module="....
```
The key parts is to specify a version on the branch module and a pkg-config entry before it with the name of the .pc file